R/path.R

```diff
@@ -62,7 +60,8 @@ path <- function(..., ext = "") {
 }
 
 #' @describeIn path_math returns the canonical path, eliminating any symbolic
-#' links.
+#'   links and the special references `~`, `~user`, `.`, and `..`, , i.e. it
+#'   calls `path_expand()` (literally) and `path_norm()` (effectively).
```

Comment:
I've been reading source and running experiments to figure out the relationships between these functions. It sure seems like `path_real()` essentially calls `path_norm()`.

Reply:
I get that now, but below there was a suggestion to "use `path_real()` prior to calling `path_norm()`" that got me really confused about which operations could be separated.